python/foglamp/services/core/api/scheduler.py

```diff
@@ -400,12 +473,13 @@ def _extract_args(data, curr_value):
         schedule = {
             'id': str(sch.schedule_id),
             'name': sch.name,
-            'processName': sch.process_name,
+            'process_name': sch.process_name,
             'type': Schedule.Type(int(sch.schedule_type)).name,
             'repeat': sch.repeat.total_seconds() if sch.repeat else 0,
             'time': (sch.time.hour * 60 * 60 + sch.time.minute * 60 + sch.time.second) if sch.time else 0,
             'day': sch.day,
-            'exclusive': sch.exclusive
+            'exclusive': sch.exclusive,
+            'enabled': sch.enabled
```

Comment:
what do you expect from client here to send in request payload for this key? 1/0, json' true/false, T/F

Reply:
same as exclusive